src/pyro/source.clj

```diff
@@ -5,9 +5,11 @@
             [glow.colorschemes :as colorschemes]
             [glow.terminal :as terminal]
             [glow.parse :as parse]
-            [instaparse.core :as insta])
+            [clojure.java.io :as io]
+            [clojure.string :as str])
   (:import [clojure.lang RT]
-           [java.io BufferedReader InputStreamReader]))
+           [java.io BufferedReader InputStreamReader]
+           (java.nio.charset StandardCharsets)))
```

Comment:
I'd prefer that we keep the styling consistent - i.e. brackets instead of braces.